kinto_client/__init__.py

```diff
@@ -169,18 +171,36 @@ def _paginated(self, endpoint, records=None, visited=None):
 
         return records.values()
 
+    def _get_cache_headers(self, safe, data=None, last_modified=None):
+        if (last_modified is None
+                and data is not None and data.get('last_modified')):
```

Comment:
nit: this identation is weird :) intermediary variable `has_data_last_modified` ?